region/norfair/crocomire/Crocomire's Room.json

```diff
@@ -775,15 +789,26 @@
       "link": [4, 4],
       "name": "Crocomire Fight (Missiles)",
       "requires": [
-        {"ammo": {"type": "Missile", "count": 25}},
         {"or": [
-          "canDodgeWhileShooting",
-          {"ammo": {"type": "Missile", "count": 25}}
+          {"and": [
+            "canDodgeWhileShooting",
+            {"ammo": {"type": "Missile", "count": 20}},
+            {"resourceMissingAtMost": [{"type": "Energy", "count": 50}]}
```

Comment:
I still prefer 25 missiles.  20 is probably most of the time but might feel bad.